src/Components/Form/Form.jsx

```diff
@@ -0,0 +1,35 @@
+import './form.css'
+export const Form = ({ thought, onNewThought, onSubmit }) => {
+  return (
+    <form onSubmit={onSubmit} netlify>
+      <label>
+        What<>&apos;</>s making you happy right now?
+        <textarea
+          rows="4"
+          cols="50"
+          value={thought}
+          onChange={onNewThought}
+          placeholder="Write some text..."
+        />
+      </label>
+      <span
+        className="wordCounter"
```

Comment:
Hey, as a heads up - even if we're doing jsx I believe the Good Code Guidelines still wants us to write class names with kebab-case (word-counter). This applies throughout your repo :)